vars/buildPlugin.groovy

```diff
@@ -63,7 +64,9 @@ def call(Map addonParams = [:])
 	]
 	def PPA_VERSION_MAP = [
 		'Matrix': [
-			'stable',
+			'stable'
```

Comment:
Why change this line?

Reply:
Also to have last value in list without a comma.

Reply:
So that the line has to be changed again, if another item is added to the Matrix list?